addons/l10n_syscohada/data/account.account.template.csv

```diff
@@ -1,1133 +1,1134 @@
-"pcg_9162","Contreparties Cautions, garanties accordées","9162","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_9163","Contreparties Hypothèques accordées","9163","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_9164","Contreparties Effets endossés par l'entité","9164","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_9168","Contreparties Autres garanties accordées","9168","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_9171","Contreparties Ventes de marchandises à terme","9171","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_9172","Contreparties Ventes à terme de devises","9172","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_9173","Contreparties Commandes fermes aux fournisseurs","9173","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_9178","Contreparties Autres engagements réciproques","9178","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_9181","Contreparties Annulations conditionnelles de dettes","9181","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_9182","Contreparties Engagements de retraite","9182","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_9183","Contreparties Achats avec clause de réserve de propriété","9183","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_9188","Contreparties Divers engagements accordés","9188","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_92","COMPTES REFLECHIS","92","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_93","COMPTES DE RECLASSEMENTS","93","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_94","COMPTES DE COUTS","94","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_95","COMPTES DE STOCKS","95","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_96","COMPTES D'ECARTS SUR COUTS PREETABLIS","96","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_97","COMPTES DE DIFFERENCES DE TRAITEMENT COMPTABLE","97","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_98","COMPTES DE RESULTATS","98","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_99","COMPTES DE LIAISONS INTERNES","99","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
\ No newline at end of file
+id,name,code,user_type_id/id,chart_template_id/id,reconcile
```

Comment:
Seems like most of the diff is because of removal of the quotes?
Please add them back.
We don't want useless diff, and it is easier to use regexps if everything has the same format

Reply:
Not really, i change the name of multiple account, cause the name wasn't clear enough

Reply:
Oh i see, i think precommit removed all the unnecessary quotes that's why the diff is on all the file

Reply:
That seems like a dumb precommit :sweat_smile:

Reply:
In any case i didn't remove all the quotes manually so a bit strange, anyway i put all the quotes back, i will try to push it again to see the diff